backend/pims.api.csproj

```diff
@@ -6,6 +6,7 @@
   </PropertyGroup>
   <ItemGroup>
     <PackageReference Include="AutoMapper.Extensions.Microsoft.DependencyInjection" Version="7.0.0" />
+    <PackageReference Include="DotNetEnv" Version="1.4.0" />
```

Comment:
Allows the `dotnet-ef` CLI to use `.env` files for configuration if we want them to.